css/css.css

```
@@ -472,4 +531,65 @@ margin-left: 100px;
 
 }
 
+/*форма*/
+.write-to-us {
+  display: flex;
+  justify-content: start;
+  align-content: flex-start;
+  flex-wrap: wrap;
+  margin-left: 240px;
+  padding-top: 63px;
+  padding-left: 100px;
+  padding-right: 99px;
+  width: 960px;
+  height: 590px;
+  background-color: #fff;
+  box-shadow: 0px 20px 40px rgba(0, 0, 0, 0.4);
+  flex-direction: column;
+  font-weight: bold;
+
+  display: noe;
+}
+
+.write-to-us-btn{
```

Если добавляешь пробел после селектора то нужно добавлять везде:
```suggestion
.write-to-us-btn {
```